compositor_render/src/transformations/web_renderer.rs

```diff
@@ -12,13 +17,18 @@ use compositor_common::{
 use log::{error, info};
 use serde::{Deserialize, Serialize};
 
-use self::{
-    browser::{BrowserClient, BrowserState},
-    chromium::ChromiumContextError,
-};
+use self::browser::{BrowserController, BrowserControllerNewError, EmbedFrameError};
+
+pub mod browser;
+pub mod chromium_context;
+mod chromium_sender;
+mod chromium_sender_thread;
+pub(crate) mod node;
+
+pub const EMBED_SOURCE_FRAMES_MESSAGE: &str = "EMBED_SOURCE_FRAMES";
+pub const UNEMBED_SOURCE_FRAMES_MESSAGE: &str = "UNEMBED_SOURCE_FRAMES";
 
-mod browser;
-pub mod chromium;
+pub const SHMEM_FOLDER_PATH: &str = "shmem";
```

Comment:
wouldn't placing this in `/tmp` make sense?

Reply:
Yeah, I will change it to tmp now